docs/configuration.md

```diff
@@ -989,7 +1002,7 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.locality.wait</code></td>
-  <td>3000</td>
+  <td>3000ms</td>
```

Comment:
Same, there are many parameters following this which are described as "in milliseconds" when they aren't in a particular unit anymore (to the caller). Consider writing values of at least 1000ms in seconds.